src/ggsn_gn.erl

```diff
@@ -237,7 +237,7 @@ handle_request(ReqKey,
 handle_request(ReqKey,
 	       #gtp{type = delete_pdp_context_request, ie = _IEs} = Request,
 	       _Resent, _State, #{left_tunnel := LeftTunnel} = Data) ->
-    ergw_gtp_gsn_lib:close_context(normal, Data),
+    ergw_gtp_gsn_lib:close_context({?MODULE, normal}, Data),
```

Comment:
I'm not too happy about this. Passing the ?MODULE like this, means it end up in the `api` label of the prometheus metric and I really don't see how that would be ok. Also, using 'gn' as API also doesn't really seem sensible.

@fholzhauser any ideas?